src/apiendpoint.ts

```diff
@@ -9,13 +9,5 @@ export namespace Apiendpoint {
   export let OSIO_ROUTE_URL: string = '';
   export const STACK_REPORT_URL: string =
```

Comment:
Is this file required anymore? It just has a bunch of empty string declarations.